packages/styles/scss/components/date-picker/_date-picker.scss

```diff
@@ -0,0 +1,213 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../colors' as *;
+@use '../../config' as *;
+@use '../../motion' as *;
+@use '../../theme' as *;
+@use '../../spacing' as *;
+@use '../../type' as *;
+@use '../../utilities/component-reset';
+@use '../../utilities/convert' as *;
+@use '../../utilities/focus-outline' as *;
+@use '../../utilities/high-contrast-mode' as *;
+@use '../../utilities/placeholder-colors' as *;
+@use '../../utilities/skeleton' as *;
+
+// I can't figure out what would've been used from this file. Safe to remove?
+// @use '../form/form';
```

Comment:
Another set of eyes on this would be appreciated! 👀  Will remove it if we can't find out why it was there in the first place.

Reply:
Sounds like the answer to this is yes, we want to continuing including these. I've pushed a fix to update.